src/anm/analytics.js

```diff
@@ -16,27 +13,15 @@ var Analytics = function () {
             var data = JSON.stringify(self.queue);
             self.queue = [];
 
-            if (supportSendBeacon) {
+            if (navigator.sendBeacon) {
                 navigator.sendBeacon(animatronUrl, data);
             } else {
-                sendViaGif(data);
+                engine.ajax(animatronUrl, null, null, 'POST', null, data, false);
             }
         }
-        if (!supportSendBeacon) {
-            setTimeout(event, timeout);
-        }
     };
 
-    var sendViaGif = function (data) {
-        if (!beacon) {
-            beacon = engine.createStatImg();
```

Comment:
...with XHR request. Please bring back `sendViaGif` and replace `engine.ajax` call with that. 

Is there any limit to amount of data that could be sent this way? May be we should optimize package size, since now we send it all at once.

Also, @RobertoUa, I think now [these options](https://github.com/Animatron/player/pull/330/files#diff-7f5481547366f17ae675b154345e696eR35) have no reason to be encrusted in every event record, they could be sent just once. To save data amount and time to send it.